modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/LinkTag.java

```diff
@@ -38,6 +38,10 @@ public int doStartTag() throws JspException {
 
 		setContainerElement("a");
 
+		if (Validator.isNotNull(_download)) {
+			setDynamicAttribute(StringPool.BLANK, "download", _download);
+		}
+
```

Comment:
We'll need to send this PR to FI for further review (unless you've already agreed on this with them).

Reply:
Oh sure, I can mention them to check this commit :D

Reply:
LGTM, just check that is being rendered correctly in React when the Tag is hydrated (if you pass some additionalProps or propsTransformer it's automatically hydrated)

Reply:
Sorry, @carloslancha I've added the two attributes but nothing is rendered by React only the plain additional attributes.

![image](https://user-images.githubusercontent.com/67901/152829067-a23c5111-fc3a-48ac-b820-12ee55b96f42.png)
![image](https://user-images.githubusercontent.com/67901/152829160-4cd3279e-0762-47b4-940f-ea5fd3f3e902.png)

Reply:
Hey, @carloslancha, did you have time to review this? It is enough to send as is now.

Reply:
@adolfopa apparently @carloslancha was mostly OK with that change

> LGTM, just check that is being rendered correctly in React when the Tag is hydrated (if you pass some additionalProps or propsTransformer it's automatically hydrated)

Did you manage to check this @boton? Do you need help?

Reply:
@julien Yep I tested here but didn't work 

> Sorry, @carloslancha I've added the two attributes but nothing is rendered by React only the plain additional attributes.
> 
> ![image](https://user-images.githubusercontent.com/67901/152829067-a23c5111-fc3a-48ac-b820-12ee55b96f42.png) ![image](https://user-images.githubusercontent.com/67901/152829160-4cd3279e-0762-47b4-940f-ea5fd3f3e902.png)

I think the hydration in react isn't merged yet liferay-frontend#1801